### PR TITLE
Classes

### PR DESCRIPTION
**Pull request by Xandarlh02**
Classes added and context added

### CODE REVIEW

File: GameEngineApi/GameEngine/Models/Game/Player.cs [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
+namespace GameEngine.Models.Game
 {
-    public class Player : User
+	
+	public class Player : User
     {
-    }
+        public int Id { get; set; }
+        public int? CurrentBet { get; set; }
+		public Dictionary<int,int>? Chips { get; set; }
```
Comment (benjaminroesdal) [inline GameEngineApi/GameEngine/Models/Game/Player.cs:12]: Should not be nullable

File: GameEngineApi/GameEngine/Models/Game/Player.cs [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+        public List<Card>? Cards { get; set; }
```
Comment (benjaminroesdal) [inline GameEngineApi/GameEngine/Models/Game/Player.cs:13]: Should not be nullable

File: GameEngineApi/GameEngine/Models/Game/User.cs [hunk 3]
```diff
@@ -2,5 +2,11 @@
 {
     public class User
     {
+        public int Id { get; set; }
+        public int Name { get; set; }
+        public string UserIdentifier { get; set; }
+        public int Wins { get; set; }
+        public Accessory Accessories { get; set; }
```
Comment (benjaminroesdal) [inline GameEngineApi/GameEngine/Models/Game/User.cs:9]: This should be a list of accessories

**Review by benjaminroesdal — APPROVED** [review]

**Review by KeanTech — APPROVED** [review]
Looks good

**Pull request merged**